docs/theme/static/js/components/upcoming-event/upcoming-event.js

```diff
@@ -0,0 +1,71 @@
+/* global moment */
+import $ from 'jquery';
+// import moment from 'moment'; // added globally
```

Comment:
I think there's a way to set the path to the CDN and just have the build "ignore" it so it wont be bundled.  cc @matthewp